### ISSUE TITLE
Why is sigma = 2?

### ISSUE THREAD

**Issue by ffineis** [acbull/Unbiased_LambdaMart#15]
Just a question about the implementation of unbiased lambdaMART - why is the sigma coefficient = 2? Is this related to numerical stability? The paper states "sigma is a constant with default value of 2" (section 4.3), but doesn't provide a reason. Most implementations of the lambdarank gradient just defaults to sigma = 1, including LightGBM. I'm just wondering what the benefits were that drove you to pick sigma = 2 as opposed to 1.

Thank you!

**Comment by acbull** [comment]
Alpha indeed a hyperparameter. It influences the shape of the sigmoid and sometimes called temperature. We just choose the one that works well on our dataset.

**Comment by ffineis** [comment]
Ha sorry by "alpha" I meant sigma. Ok got it, thanks much.
